teachertool/src/state/actions.ts

```diff
@@ -41,8 +53,19 @@ const removeNotification = (notificationId: string): RemoveNotification => ({
     notificationId,
 });
 
+const setProjectMetadata = (meta: pxt.Cloud.JsonScript | undefined): SetProjectMetadata => ({
```

Comment:
nit: not a fan of `meta` as a parameter name, is there a reason we don't have it be `metadata`? Then we could just have the object have `type: "xyz", metadata`, right?